### ISSUE TITLE
Type error: Cannot find module '@/components/layout/rootLayout' or its corresponding type declarations.

### ISSUE THREAD

**Issue by etmorefish** [etmorefish/AIGC#8]
I'm having issues with multiple components like this. Locally on my computer when I run "npm run build", it works fine.

I've been using node version 18 locally, but no luck. I've never had this issue before.

![image](https://github.com/etmorefish/AIGC/assets/26000187/8e47e420-41be-4f6a-88af-59b6c1ab2e0c)

**Comment by etmorefish** [comment]
The 'module not found' error is a syntax error that appears when the static import statement cannot find the file at the declared path. This common syntax error is caused by letter-casing inconsistencies that are present in your filename(s) between your repository and local machine, or both.

# [Why this Happens](https://vercel.com/guides/how-do-i-resolve-a-module-not-found-error#why-this-happens)
One possible cause for this issue is the fact that some filesystems are case-insensitive, however Vercel deployments use a case-sensitive filesystem. Because of this, it is possible that when you change the letter-casing in filenames on your local machine, Git will only pick up changes in your respective static import statements. This would mean that your repository is now syntactically incorrect, resulting in a build error when deploying the repository on the Vercel platform.

# [How to Resolve](https://vercel.com/guides/how-do-i-resolve-a-module-not-found-error#how-to-resolve)
The [build logs](https://vercel.com/knowledge/where-are-vercel-logs) will indirectly point to the file that is the causing the issue. Make sure that your file's letter-casing correctly matches with its respective static import. Also make sure that your letter-casing in your filenames and imports are identical between your repository and local machine. If you are using git, make sure git config core.ignorecase false is set in your environment.

This answer worked for me: https://stackoverflow.com/a/55541435/3051080

TL;DR; update `git cache`:
```
git rm -r --cached .
git add --all .
git commit -a -m "Versioning untracked files"
git push origin master
```

**Comment by etmorefish** [comment]
If you are using git, make sure `git config core.ignorecase false` is set in your environment.
